media-autobuild_suite.bat

```diff
@@ -423,7 +423,8 @@ if %svthevcINI%==0 (
     echo. 1 = Yes
     echo. 2 = No
     echo.
-    echo. Requires at least an Intel first generation core or AMD Bulldozer.
+    echo. Note: Requires at least an Intel first generation core or AMD Bulldozer to
+    echo. build and run. This requires sse4.1 extensions.
```

Comment:
Wow, even to build?

Reply:
Based on my testing, yes. Although my testing may be flawed based on my knowledge with gcc.

Reply:
I could easily (Hopefully) be wrong.